wormchain/x/wormhole/keeper/query_plugins.go

```diff
@@ -57,8 +58,8 @@ func WormholeQuerier(keeper Keeper) func(ctx sdk.Context, data json.RawMessage)
 		}
 		if wormholeQuery.VerifySignature != nil {
 			// handle the verify signature query
-			digest := vaa.SigningMsg(wormholeQuery.VerifySignature.Data).Bytes()
-			err := keeper.VerifySignature(ctx, digest, wormholeQuery.VerifySignature.GuardianSetIndex, wormholeQuery.VerifySignature.Signature)
+			digest := ethCrypto.Keccak256Hash(wormholeQuery.VerifySignature.Data)
```

Comment:
I think this was written to avoid doing the hash in cosmwasm, but I don't think we're close to bottlenecking in performance there, so I can apply this change.